bentoml/_internal/yatai_client/config.py

```diff
@@ -0,0 +1,82 @@
+import os
+from pathlib import Path
+from typing import List
+
+import attr
+import cattr
+import yaml
+
+from bentoml._internal.yatai_client.yatai import YataiClient
+from bentoml._internal.yatai_client.errors import YataiCliError
+
+default_context_name = 'default'
+
+
+def get_config_path() -> Path:
+    return Path.home() / '.yatai.yaml'
+
+
+@attr.define
+class YataiClientContext:
+    name: str
+    endpoint: str
+    api_token: str
+
+    def get_yatai_cli(self) -> YataiClient:
```

Comment:
rename to `get_yatai_client`? it might be a bit confusing since  `cli` usually means `command-line` by convention especially with python and click